### ISSUE TITLE
vs-upload response data

### ISSUE THREAD

**Issue by smccoy78** [lusaxweb/vuesax#856]
Is it possible to get the data returned in the response after upload success?

**Comment by yashtibrewal** [comment]
+1

**Comment by kacou12** [comment]
Yes it is possible, by putting a ref attribute on the vs upload tag, example: ref = "uploadImg"
Then in some method, we get it with  this.$refs.uploadmg.filesx
